arcgis-ios-sdk-samples/Analysis/Statistical query (group and sort)/StatisticalQueryGroupAndSortViewController.swift

```diff
@@ -0,0 +1,424 @@
+    
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var getStatisticsButton: UIBarButtonItem!
+    @IBOutlet private weak var tableView: UITableView!
+    private var serviceFeatureTable: AGSServiceFeatureTable!
+    private var fieldNames = [String]()
+    private var selectedGroupByFieldNames = [String]()
+    private var orderByFields = [AGSOrderBy]()
+    private var selectedOrderByFields = [AGSOrderBy]()
+    private var statisticDefinitions = [AGSStatisticDefinition]()
+    private var sectionHeaderTitles = ["1. Add Statistic Definitions", "2. Select Group By Fields", "3. Select Order By Fields"]
+    private var statisticTypes = ["Average", "Count", "Maximum", "Minimum", "StandardDeviation", "Sum", "Variance"]
+    private weak var expandableTableViewController: ExpandableTableViewController!
+    private weak var groupByFieldsViewController: GroupByFieldsViewController!
+    private weak var orderByFieldsViewController: OrderByFieldsViewController!
+    private weak var addStatisticDefinitionsViewController: AddStatisticDefinitionsViewController!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Add the source code button item to the right of navigation bar
+        (navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["StatisticalQueryGroupAndSortViewController", "AddStatisticDefinitionsViewController", "GroupByFieldsViewController", "OrderByFieldsViewController"]
+        
+        // Initialize feature table
+        serviceFeatureTable = AGSServiceFeatureTable(url: URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer/3")!)
+        
+        // Load feature table
+        serviceFeatureTable.load(completion: { [weak self] (error) in
+            //
+            // If there an error, display it
+            guard error == nil else {
+                SVProgressHUD.showError(withStatus: error!.localizedDescription, maskType: .gradient)
+                return
+            }
+            
+            // Set title
+            let tableName = self?.serviceFeatureTable.tableName
+            self?.titleLabel.text = "Statistics: \(tableName ?? "")"
+
+            // Get field names
+            for field in (self?.serviceFeatureTable.fields)! {
+                self?.fieldNames.append(field.name)
+            }
+        })
+        
+        // Setup UI Controls
+        setupUI()
+    }
+    
+    private func setupUI() {
+        //
+        // Set corner radius and border for tables
+        tableView.layer.cornerRadius = 10
+        tableView.layer.borderWidth = 1
+        tableView.layer.borderColor = UIColor.lightGray.cgColor
+        
+        // Register table cell
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "TableViewCell")
```

Comment:
I think this line isn't necessary as it's registered in the Storyboard already.

Reply:
fixed